l10n_it_invoices_data_communication/models/communication.py

```diff
@@ -43,6 +44,16 @@ def check_normalized_string(value):
     return normalized
 
 
+def normalize_encode(value, form='NFKD', encoding='ascii', errors='replace'):
+    try:
+        # Soluzione proposta in PR #1505
+        # res = value.encode('latin', 'replace').decode('latin')
+        res = unicodedata.normalize(form, value).encode(encoding=encoding, errors=errors)
+    except Exception:
```

Comment:
Non esiste un'eccezione più specifica che copra questo caso?

Reply:
L'eccezione generata in realtà era TypeError che si verificava quando si effettua il normalize di una stringa vuota. Ho fatto un controllo prima e ho rimosso del tutto il try/excerpt